src/otx/core/model/utils/mmpretrain.py

```diff
@@ -48,3 +54,132 @@ def create_model(config: DictConfig, load_from: str | None = None) -> tuple[nn.M
     """
     classification_layers = get_classification_layers(config, MODELS, "model.")
     return build_mm_model(config, MODELS, load_from), classification_layers
+
+
+class ForwardExplainMixInForMMPretrain(Generic[T_OTXBatchPredEntity, T_OTXBatchDataEntity]):
```

Comment:
@negvet I wonder if we can come up with better naming, because it seems not really clear to me. How about `ExplainableMMPretrainModel`?

Reply:
This is a mixin class that cannot be working standalone. Therefore, I think that it should include `MixIn` keyword in its name such as `ExplainableMixInMMPretrain`. I'm open to the new naming but hope `MixIn` will be in it.

Reply:
Thank you for this clarification :) How about `ExplainableMixInMMPretrainModel`?